sammba/externals/nipype/interfaces/afni/utils.py

```diff
@@ -1561,6 +1561,9 @@ class NwarpApplyInputSpec(CommandLineInputSpec):
         desc='be extra verbose :)',
         argstr='-verb',
         xor=['quiet'])
+    newgrid = traits.Float(
+        argstr='-newgrid %f',
+        desc='Write the output dataset using isotropic grid spacing in mm.')
```

Comment:
>Does it have to be isotropic

Yes, because only one float is allowed.

>the edits in func.py seem to allow non-isotropic voxels

You are right. But actually I am no more using this `newgrid` parameter, I am manually resampling the taget template to the chosen `voxel_size`. So may be I am going just to remove the parameter `newgrid` because it is no longer used

Reply:
in which case "voxel_size : 3-tuple of floats, optional " will need to be changed in the definition help/instructions. that said resample() needs a tuple, and I can't see where in the code voxel_size gets turned into a tuple before being passed on to resample()

Reply:
a bizarre option, though one that retains maximum flexibility and reduces the lines of code to write, is to continue requiring the user to pass a tuple. this will almost always be of the same three numbers (I can't think of a use case where anisotropic would be useful), though you never know, someone might need individual control of each one